src/components/SideBySide/SideBySide.vue

```diff
@@ -0,0 +1,130 @@
+<template lang="html">
+    <div v-if="!isMobile" class="side-by-side">
+        <div class="images">
+            <div
+                v-for="(image, i) in images"
+                v-if="images.length"
+                class="wrapper"
+            >
+                <wp-image
+                    v-if="image"
+                    :key="image.id"
+                    :image="image"
+                    :aspect-ratio="130"
+                />
+            </div>
+        </div>
+        <div class="text">
+            <wp-content v-for="(text, i) in texts" :key="text" :html="text" />
+        </div>
+    </div>
+    <div v-else class="side-by-side">
+        <div v-for="(item, i) in items" v-if="items.length" class="mobile-item">
+            <wp-image
+                v-if="item.image"
+                :key="item.image.id"
+                :image="item.image"
+                :aspect-ratio="130"
+            />
+            <wp-content :key="item.text" :html="item.text" />
+        </div>
+    </div>
+</template>
+
+<script>
+import WpImage from "@/components/global/WpImage"
+import WpContent from "@/components/global/WpContent"
+
+export default {
+    components: {
+        WpImage,
+        WpContent
+    },
+    props: {
+        items: {
+            type: Array,
+            default: () => []
+        },
+        isMobile: {
+            type: Boolean,
+            default: false
+        }
+    },
+    computed: {
+        images() {
+            let images = []
+            this.items.forEach(el => images.push(el.image))
+            return images
+        },
+        texts() {
+            let text = []
+            this.items.forEach(el => text.push(el.text))
+            return text
+        }
+    }
+}
+</script>
+
+<style lang="scss">
+.side-by-side {
+    background-color: beige; //DELETE placeholder
```

Comment:
Remove this. I think you can style this in the `story` perhaps? Would be a good way to solve this issue.

Also you should have a `--unit-100vh` min-height on this I think.

Reply:
@dChiamp you didn't remove the placeholder color.